jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/view.js

```diff
@@ -61,6 +61,9 @@ ORYX.Plugins.View = {
         this.maxZoomLevel = 2.0;
         this.diff=5; //difference between canvas and view port, s.th. like toolbar??
 
+        // source view window
+        this.sourcewin;
+
```

Comment:
@tsurdilo it's not needed. you're not defining anything either as it's not being initialized.

this.foo;
alert(this.foo);

this.foo is undefined, of course!